Source/WebCore/Modules/webcodecs/WebCodecsAudioDecoder.cpp

```diff
@@ -160,25 +160,27 @@ ExceptionOr<void> WebCodecsAudioDecoder::decode(Ref<WebCodecsEncodedAudioChunk>&
         m_isKeyChunkRequired = false;
     }
 
-    ++m_decodeQueueSize;
```

Comment:
According spec, we should probably increment the operation queue, I understand this is in queueControlMessageForCodecOperationAndProcess but this seems more confusing to have both queueControlMessageForCodecOperationAndProcess and queueControlMessageProcess. I suggest removing queueControlMessageForCodecOperationAndProcess.